src/bors/command/mod.rs

```diff
@@ -53,4 +55,6 @@ pub enum BorsCommand {
     Delegate,
     /// Revoke any previously granted delegation.
     Undelegate,
+    /// Rollup status.
+    Rollup(String),
```

Comment:
`rollup` is pretty much just four states (never/iffy/maybe/always), so it would be nicer to model this with an enum.